src/System.IO.FileSystem/tests/Directory/EnumerableTests.cs

```diff
@@ -31,6 +31,32 @@ public void FileEnumeratorIsThreadSafe()
             {
             }
         }
+        
+        [Fact]
+        public void EnumerateDirectories_NonBreakingSpace()
+        {
+            string curDir = Directory.GetCurrentDirectory();
+            DirectoryInfo di = new DirectoryInfo(curDir + @"\\Temp");
```

Comment:
This should use GetTestFilePath() rather than creating directories/files under the execution directory.